dynamic_offset_hip: -0.005
dynamic_offset_knee: 0.009999999999999998
dynamic_offset_ankle: 0